lib/workers/repository/update/branch/check-existing.spec.ts

```diff
@@ -19,13 +19,13 @@ describe('workers/repository/update/branch/check-existing', () => {
     });
 
     it('returns false if recreating closed PRs', async () => {
-      config.recreateClosed = true;
+      config.recreateClosed = 'always';
       expect(await prAlreadyExisted(config)).toBeNull();
       expect(platform.findPr).toHaveBeenCalledTimes(0);
     });
 
     it('returns false if check misses', async () => {
```

Comment:
The test name doesn't explain _which_ check is needed. Also `misses` is a bit weird, I think we mean "is not found".

Something like this explains what the test does:

```ts
it('returns false if name_of_check check is not found', async () => {
```

Reply:
the describe block says that we are checking for fn `prAlreadyExisted` ie. the check must be for checking existing pr...
"misses" meaning that `prAlreadyExisted()` couldn't retrieve desired result ie. info on a existing PR

Seems okay tbh